### ISSUE TITLE
Some test failures not detected by Teamcity (no output from NUnit)

### ISSUE THREAD

**Issue by oskarb** [nunit/nunit-console#668]
Nunit console 3.10. Some test failures are reported in the output from each test as each test finishes but failures from other test cases are only reported in the summary at the end after all tests have finished. Parallel execution not used. Failures that are only reported in the summary causes Teamcity to not detect these failures.

The failures that are reported for each affected test case actually occurs in OneTimeSetup. The failures (for other test cases) that are only reported in the summary occur in regular [Setup], or in the test case itself.

What can cause this?

I have create a bug for Teamcity here, where I provided sample log output to show the problem:
https://youtrack.jetbrains.com/issue/TW-61442

**Comment by CharliePoole** [comment]
Make sure you have the latest version of the teamcity extension for NUnit, 1.0.6.

**Comment by CharliePoole** [comment]
@ChrisMaddock It seems like it would be useful if the `--list-extensions` option gave the version of each extension present.

**Comment by oskarb** [comment]
> Make sure you have the latest version of the teamcity extension for NUnit, 1.0.6.

Hmm, an interesting suggestion. I have the latest version of Teamcity (currently 2019.1.2 (build 66342)) and I have used its Tools menu to have TC itself download and install NUnit Console 3.10. I can verify that this in fact seems to not have installed the latest version of the teamcity extension, but instead uses 1.0.4.

**Comment by CharliePoole** [comment]
It's likely that 1.0.4 was the latest version available at the time 3.10 was released. When a new version of NUnit.Console is released, it will include the latest available extensions, but you can also update them yourself at any time.

**Comment by oskarb** [comment]
1.0.6 is in fact marked as being released 4 days before 3.10, and the NUnit console Nuget package depends on 1.0.6 but it seems to have not been updated in the zip-file. With the way it's setup in teamcity it doesn't really seem like I am supposed to have to fiddle with the extensions dll manually, but I guess I could try it just to see if it makes a difference. Not sure where to find it though - since there is no zip-file with the correct version included it seems the only way is to extract it from the nuget package.

**Comment by oskarb** [comment]
Actually it's even more messed up.

When I download this package from nuget:
nunit.extension.teamcityeventlistener.1.0.6.nupkg

The file and product version of the teamcity-event-listener.dll:
1.0.4.0

It seems likely that the DLL contents is correct but the version number has not been updated properly. @NikolayPianikov 

I've compared to the extension DLL from the console zip-file, and also to the extension DLL actually found in the teamcity agent. These binary files are exactly the same.

So what I can say for sure is that I am already using the teamcity-event-listener.dll from 1.0.6 of the nuget package, even though the file itself claims to be 1.0.4.

**Comment by oskarb** [comment]
@NikolayPianikov PR to correct version number in AssemblyInfo.cs (just a temporary solution to raise awareness): https://github.com/nunit/teamcity-event-listener/pull/69

**Comment by CharliePoole** [comment]
@ChrisMaddock Shouldn't this be transferred to the extension project?

**Comment by ChrisMaddock** [comment]
Yep, I agree. @oskarb - it looks like the console is reporting correctly here, and the issue is instead with the TeamCity Extension. I'll move this over now, @NikolayPianikov can always move it back if we have reason to believe otherwise.
